reference/api/keys.md

```diff
@@ -232,15 +226,11 @@ Date and time when the key will expire, represented in RFC 3339 format. `null` i
 
 <RouteHighlighter method="PATCH" route="/keys/:key"/>
 
-Update the description, permissions, or expiration date of an API key. To learn more about the variables sent in the body of the request, see the [create key](#body) endpoint.
+Update the description, permissions, or expiration date of an API key. The [`key`](/reference/api/keys.md#key) is required.
```

Comment:
Not sure if "the `key`" is the right term here. Would "`key` value" be better?

Reply:
Perhaps we can break the pattern and say:

```suggestion
Update the description, permissions, or expiration date of an API key. A valid API [key](/reference/api/keys.md#key) is required.
```

Reply:
Perhaps we can break the pattern and say:

```suggestion
Update the description, permissions, or expiration date of an API key. A valid API [key](/reference/api/keys.md#key) is required.
```

Reply:
This is better, thanks!